frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelServingGlobal.cy.ts

```diff
@@ -224,17 +237,29 @@ describe('Model Serving Global', () => {
     inferenceServiceModal.findSubmitButton().should('be.disabled');
   });
 
-  it('All projects loading', () => {
-    initIntercepts({ delayInferenceServices: true, servingRuntimes: [], inferenceServices: [] });
+  it('All projects loading and cancel', () => {
+    initIntercepts({
+      delayInferenceServices: true,
+      delayServingRuntimes: true,
```

Comment:
Adding arbitrary delays seems like a bad idea by extending test execution time as well as potential for flakes if you rely on something to happen before or after the delay without any concrete constraint.

I added support to the snapshot cypress tests utils to give better control of when to trigger a response. I'll have to look into adding similar support for general use.

Reply:
@christianvogt Yes, that's a great point. I was concerned by this same thing but didn't found any other way to test this scenario. What's being tested here is that if the user navigates to other namespace, the request is aborted, and for that i needed a little bit of delay.

Having control of triggering a response is great, but im not even sure if that's what i want. Do you have something in mind tha could fit my case here? (btw, i think the delay is way too long, i can decrease it as a middle ground if we don't find anything suitable to this scenario)

Reply:
@lucferbux i think the best thing to do is add a `TODO` around the delay. Then also log a bug for us to add tooling support for your use case without delays. Include use case details and any nuances.

Reply:
Ok, great thanks for the input! I'm gonna do it right away!